integration/update_field_compat_test.go

```diff
@@ -54,6 +56,85 @@ func TestUpdateFieldCompatInc(t *testing.T) {
 	testUpdateCompat(t, testCases)
 }
 
+func TestUpdateFieldCompatMax(t *testing.T) {
+	t.Parallel()
+
+	testCases := map[string]updateCompatTestCase{
+		"Int32Lower": {
+			update: bson.D{{"$max", bson.D{{"v", int32(30)}}}},
+		},
+		"Int32Higher": {
+			update: bson.D{{"$max", bson.D{{"v", int32(60)}}}},
+		},
+		"Int32Negative": {
+			update: bson.D{{"$max", bson.D{{"v", int32(-22)}}}},
+		},
+		"Document": {
+			update: bson.D{{"$max", bson.D{{"v", bson.D{{"foo", "bar"}}}}}},
+			skip:   "https://github.com/FerretDB/FerretDB/issues/457",
+		},
+		"EmptyDocument": {
+			update: bson.D{{"$max", bson.D{{"v", bson.D{{}}}}}},
+			skip:   "https://github.com/FerretDB/FerretDB/issues/1000",
+		},
+		"Double": {
+			update: bson.D{{"$max", bson.D{{"v", 54.32}}}},
+		},
+		"DoubleNegative": {
+			update: bson.D{{"$max", bson.D{{"v", -54.32}}}},
+		},
+
+		"MultipleQueries": {
+			update: bson.D{{"$max", bson.D{{"v", int32(39)}, {"a", int32(30)}}}},
```

Comment:
Do I understand correctly that a test case where we set `$max` to a field that doesn't exist is covered by this case too?

I mean something like
```
update: bson.D{{"$max", bson.D{{"a", int32(30)}}}},
```

Reply:
It looks like it is (we compare modifiedCount and collection values) but to be more detailed I can add another test case. It will be better for debugging. Thanks!